projects/plugins/jetpack/changelog/add-move-woo-event-tracking-frontend

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: other
+Comment: We updated how we track Cart/Checkout
```

Comment:
This is because I based my PR on https://github.com/Automattic/jetpack/pull/35139